src/levanter/compat/hf_checkpoints.py

```diff
@@ -477,10 +478,15 @@ def load_pretrained(
 
             # Vocab: next, we resize the desired actual size
             if Vocab.size != tokenizer_Vocab.size:
-                logger.info(
-                    f"Resizing model from {Vocab.size} to {tokenizer_Vocab.size} to match tokenizer vocab size"
-                )
-                lev_model = lev_model.resize_vocab(tokenizer_Vocab.size)
+                if resize_vocab_to_match_tokenizer:
```

Comment:
In which case do we want to keep the vocab size different from tokenizer? @dlwh

Reply:
this is a good point. I was thinking it would mean keep whatever's in the model itself? WDYT?